components/brave_rewards/browser/rewards_service.h

```diff
@@ -76,6 +77,13 @@ using GetPublisherBannerCallback =
     base::OnceCallback<void(std::unique_ptr<brave_rewards::PublisherBanner>)>;
 using RefreshPublisherCallback =
     base::OnceCallback<void(uint32_t, const std::string&)>;
+using GetPublisherInfoCallback = base::OnceCallback<void(
```

Comment:
could this two be merged and just use `GetPublisherInfoCallback` as signatures are the same

Reply:
Yes, done.